src/cpp/shared/resource/resourceManager.h

```diff
@@ -76,6 +76,8 @@ class resourceManager : baseManager
 
 extern resourceManager* gpResourceManager;
 
+extern void __fastcall WaitEndSample(struct SAMPLE2, int);
```

Comment:
I'm not actually sure about the specifics of the workings of this function--nevermind, I just figured it out, I think. It looks like it is used to factor in some game delays (or lag?) in the consideration of when to poll sounds in-game. I see why this would be in sound.h, but I was never given the reasoning as to why Kert wanted it in resourceManager.h, so I guess I'll run it by him, too